application-templates/javascript/connect.yaml

```diff
@@ -54,4 +54,15 @@ deployAs:
         - key: CTP_CLIENT_SECRET
           description: commercetools Composable Commerce client secret
         - key: CTP_SCOPE
-          description: commercetools Composable Commerce client scope
\ No newline at end of file
+          description: commercetools Composable Commerce client scope
+  - name: mc-app
+    applicationType: mc-app
+    endpoint: /
+    configuration:
+      standardConfiguration:
+        - key: APPLICATION_ID
+          description: commercetools Composable Commerce application ID
+          required: true
+        - key: CLOUD_IDENTIFIER
+          description: commercetools Composable Commerce cloud identifier
```

Comment:
We can set the description similar to Merchant Center to make it clearer.
`The identifier of the cloud Region that the Custom Application uses to connect to Composable Commerce. The values map to the actual Merchant Center API URL for that Region.`

More details can be found here: https://docs.commercetools.com/custom-applications/api-reference/application-config#cloudidentifier